packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -110,6 +111,14 @@ Rect _globalBoundingBoxFor(BuildContext context) {
 /// B to A, route A's hero's widget is, by default, placed over where route B's
 /// hero's widget was, and then the animation goes the other way.
 ///
+/// ## Nested Navigators
```

Comment:
The previous header only makes sense if there was only one header. We should refactor the doc a bit and split it up if there's more than one.

Reply:
What's your suggestion? I thought of "Nested Navigators" as just a sub-topic of what's there to Discuss on this Widget (I realize now it should have `###` instead of just `##`).

Reply:
### is fine